src/command/search.rs

```diff
@@ -169,6 +192,16 @@ async fn key_handler(
                     .map_or(app.input.clone(), |h| h.command.clone()),
             );
         }
+        Key::Alt(c) if ('1'..='9').contains(&c) => {
```

Comment:
I've thought of an even better way of doing this :D

```suggestion
        Key::Alt('1'..='9') => {
```

Rust pattern matching is so powerful

Reply:
If you want to read up more about patterns, see https://doc.rust-lang.org/book/ch18-03-pattern-syntax.html

Reply:
@conradludgate if I use the range in the pattern, how do I get access to `c`?

Reply:
Hmm. Good point. I think maybe

`Key::Alt(c @ '0'..='9')`

Reply:
While this is cool, I think for the sakes of clarity the current approach might be best. Contributors new to Rust (and many not new to Rust) might find it confusing or unclear

Reply:
[(Rust is crazy! 🤣)](https://doc.rust-lang.org/book/ch18-03-pattern-syntax.html#-bindings)

Reply:
For now, ignore this. I think I'll go in for a round 2 cleaning up some code later. I also want to start working on a minimum supported rust version and getting that enforced